Code/GraphMol/FMCS/Wrap/rdFMCS.cpp

```diff
@@ -47,6 +47,15 @@ void SetMCSBondTyper(MCSParameters &p, BondComparator bondComp) {
   }
 }
 
+ROMol *getQueryMol(MCSResult &mcsRes) {
```

Comment:
Why not just return QueryMol directly? Why require the copy?

Reply:
@greglandrum Of course! I have no idea.